build_projects/update-dependencies/update-dependencies.ps1

```diff
@@ -17,20 +17,18 @@ if($Help)
     exit 0
 }
 
-$Architecture='x64'
-
 $RepoRoot = "$PSScriptRoot\..\.."
 $AppPath = "$PSScriptRoot"
 
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows\$Architecture"
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows"
```

Comment:
I think we want the "Windows" portion of this out of the path, but we want to keep the $Architecture piece.  That way we can build/test x64 and x86 in the same repo without cleaning in between.